service/src/main/kotlin/xyz/lebkuchenfm/domain/radiopersonality/RadioPersonalityService.kt

```diff
@@ -38,6 +38,16 @@ class RadioPersonalityService(
 
         return RadioPersonalityUtterance(text, audio)
     }
+
+    suspend fun userControlledUtterance(userMessage: String): RadioPersonalityUtterance? {
+        val audio = textToSpeechProvider.synthesize(userMessage)
+            ?: run {
+                logger.error { "Could not synthesize the text to audio." }
```

Comment:
now we log almost same error twice: here and in command processor

Reply:
That's not true. Here we provide specific reason for the error. In command processor we have generic message that we're sending back to Discord.